supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc

```diff
@@ -728,6 +740,17 @@ In IdM, you can also create multiple *sub-CAs*. Sub-CAs are IdM CAs whose certif
 
 *See also*: xref:bucket[bucket], xref:connection[connection]
 
+[discrete]
+[[control-plane]]
+==== image:images/yes.png[yes] control plane (noun)
+*Description*: In Red Hat OpenShift, a "control plane" is a container orchestration layer that exposes the API and interfaces to define, deploy, and manage the lifecycle of containers. Also known as control plane machines.
```

Comment:
Same comment about using italics for _control plane_. Also I'd suggest "the" control plane, since there is just one per cluster.

I would consider removing the sentence about "also known as control plane machines", because that's not quite true. There are control plane machines or nodes or other components that follow "control plane", which is appropriate depends on the situation sometimes.